tests/phpunit/elementor/core/upgrade/test-upgrades.php

```diff
@@ -453,4 +477,4 @@ private function create_image() {
 	private function delete_image( $attachment_id ) {
 		wp_delete_attachment( $attachment_id, true );
 	}
-}
+}
```

Comment:
Blank line?

Reply:
@louiswol94 
"phpcs: File must end with a newline character"

Reply:
@TzviRabinovitch Yes I asked you to add the blank line. In the diff it showed you deleted the blank line. Sorry if I was unclear.